features/wishlists.feature

```diff
@@ -26,6 +26,37 @@ Scenario: Create a wishlist
     And the "Wishlist_Name" field should be empty
     And the "Wishlist_User_Id" field should be empty
     When I paste the "Wishlist_Id" field
-    And I press the "Search-Wishlist" button
+    And I press the "retrieve-Wishlist" button
     Then I should see "Jan Wishlist" in the "Wishlist_Name" field
     And I should see "1" in the "Wishlist_User_Id" field
+
+Scenario: List all wishlists
+    When I visit the "Home Page"
+    And I press the "Search-wishlist" button
+    Then I should see "My wishlist" in the results
+    And I should see "My greatest wishlist" in the results
+
+Scenario: Rename a wishlist
+    When I visit the "Home Page"
+    And I press the "Search-wishlist" button
+    Then I should see "My wishlist" in the results
+    When I copy the "Wishlist_Id" field
```

Comment:
Oh this was my mistake. I believe we should keep everything in `kebab-case` for classnames and ids. If you have time, could you modify all these to be `wishlist-id`, `wishlist-name`, etc.? If not, I can fix this in my PR.